Document automatic developer mode activation (#17)
fiskaltrust/possystemapi-devkit: README.MD

**IMPORTANT**: This feature is available with InStore App v1.2.8-rc1 and later.

The InStore App supports a developer mode that exposes a hidden **Dummy Payment Provider**, allowing easy integration and testing of different payment success and error scenarios without requiring access to a real payment provider.

> Developer mode is now activated automatically based on the type of cashbox the InStore App is paired with:

Reviewer: ```suggestion
Developer mode is now automatically activated based on the type of CashBox that the InStore App is paired with:
```

> - **Sandbox cashbox** — Developer mode is **enabled** automatically.

Reviewer: ```suggestion
- **Sandbox CashBox** - Developer mode is **enabled** automatically.
```

> - **Production cashbox** — Developer mode is **disabled** automatically.

Reviewer: ```suggestion
- **Production CashBox** - Developer mode is **disabled** automatically.
```

> There is no manual toggle, no logo tap, and no PIN entry required.

Reviewer: ```suggestion
Manual controls are no longer available: there is no toggle, logo tap, or PIN entry required.
```

> ## Configuring the Dummy Payment Provider

Reviewer: ```suggestion
### Configuring the Dummy Payment Provider
```

> 1. Make sure the InStore App is paired with a **sandbox** cashbox.

Reviewer: ```suggestion
1. Ensure the InStore App is paired with a **sandbox** CashBox.
```

> 2. Open **Settings** and navigate to the **Payment settings** section.

Reviewer: ```suggestion
2. Open **Settings** and navigate to the **Payment Settings** section.
```

3. Tap **Payment entry** (the first item in **Payment Settings**) and select the **Dummy Payment Provider**, which is visible because developer mode is active.

> **Using the Dummy Payment Provider**

Reviewer: ```suggestion
### Using the Dummy Payment Provider
```
